docker-extras/docker-compose/ckan-conf-templates/datagov-theme-production.ini.template

```diff
@@ -0,0 +1,232 @@
+[DEFAULT]
+debug = false
+
+[server:main]
+use = egg:Paste#http
+host = 0.0.0.0
+port = 5000
+
+[app:main]
+use = egg:ckan
+full_stack = true
+cache_dir = /tmp/%(ckan.site_id)s/
+beaker.session.key = ckan
+
+beaker.session.secret = {{BEAKER_SESSION_SECRET}}
+app_instance_uuid = {{APP_INSTANCE_UUID}}
+
+who.config_file = %(here)s/who.ini
+who.log_level = warning
+who.log_file = %(cache_dir)s/who_log.ini
+# Session timeout (user logged out after period of inactivity, in seconds).
+# Inactive by default, so the session doesn't expire.
+# who.timeout = 86400
+
+## Database Settings
+sqlalchemy.url = {{SQLALCHEMY_URL}}
+
+ckan.datastore.write_url = {{CKAN_DATASTORE_WRITE_URL}}
+ckan.datastore.read_url = {{CKAN_DATASTORE_READ_URL}}
+
+# PostgreSQL' full-text search parameters
+ckan.datastore.default_fts_lang = english
+ckan.datastore.default_fts_index_method = gist
+
+
+## Site Settings
+
+ckan.site_url = http://ckan:5000
+#ckan.use_pylons_response_cleanup_middleware = true
+
+## Authorization Settings
+
+ckan.auth.anon_create_dataset = false
+ckan.auth.create_unowned_dataset = false
+ckan.auth.create_dataset_if_not_in_organization = true
+ckan.auth.user_create_groups = false
+ckan.auth.user_create_organizations = false
+ckan.auth.user_delete_groups = false
+ckan.auth.user_delete_organizations = false
+ckan.auth.create_user_via_api = false
+ckan.auth.create_user_via_web = true
+ckan.auth.roles_that_cascade_to_sub_groups = admin
+
+
+## Search Settings
+
+ckan.site_id = default
+solr_url = {{SOLR_URL}}
+
+
+## Redis Settings
+
+# URL to your Redis instance, including the database to be used.
+ckan.redis.url = {{CKAN_REDIS_URL}}
+
+
+## CORS Settings
+
+# If cors.origin_allow_all is true, all origins are allowed.
+# If false, the cors.origin_whitelist is used.
+# ckan.cors.origin_allow_all = true
+# cors.origin_whitelist is a space separated list of allowed domains.
+# ckan.cors.origin_whitelist = http://example1.com http://example2.com
+
+
+## Plugins Settings
+
+#		Add ``resource_proxy`` to enable resorce proxying and get around the
+#		same origin policy
+
+ckan.plugins = datastore 
+    stats text_view image_view recline_view 
+    xloader archiver report structured_data 
+    datajson_harvest datajson datagovtheme
+    dcat dcat_rdf_harvester dcat_json_harvester dcat_json_interface 
+    harvest ckan_harvester 
+    spatial_metadata spatial_query spatial_harvest_metadata_api
+    geodatagov geodatagov_miscs z3950_harvester 
+    arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection 
+    geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester
```

Comment:
do we need all these plugins in the test environment?

Reply:
Cleaned [here](https://github.com/datopian/ckan-ng-harvest/commit/42623ed8bd31fee525c8a9c8c4e89166606ac47e)